nengo_viz/static/viz.js

```diff
@@ -127,6 +139,18 @@ VIZ.Component.prototype.on_resize = function(width, height) {};
  */
 VIZ.Component.prototype.on_message = function(event) {};
 
+
+VIZ.Component.prototype.generate_menu = function() {
+    var self = this;
+    var items = [];
+    if (this.label_visible) {
+        items.push(['hide label', function() {self.hide_label();}]);
+    } else {
+        items.push(['show label', function() {self.show_label();}]);    
+    }
+    return items;
```

Comment:
items becomes a single 1 item array with an array inside i.e.
[['show label', function()...]]

Not sure if that is your intended effect...